bus-mapping/src/evm/opcodes/sstore.rs

```diff
@@ -33,6 +33,15 @@ impl Opcode for Sstore {
                 value: Word::from(state.tx_ctx.id()),
             },
         );
+        state.push_op(
```

Comment:
Looks good overall!

I think we should also assert the `CallContext` lookups being added in this bus-mapping implementation, along with the `StackOp` and `StorageOp` checks.

Reply:
you mean assert `CallContext` ops in below unit test  ?

Reply:
yes

Reply:
updated in https://github.com/appliedzkp/zkevm-circuits/commit/7e7470ad9b0a7a602e0348d70c81c65e59a57cc2 thanks